diff --git a/data/transactions/verify/txn_test.go b/data/transactions/verify/txn_test.go
index 7e3015ef0..536544d8f 100644
--- a/data/transactions/verify/txn_test.go
+++ b/data/transactions/verify/txn_test.go
@@ -64,7 +64,7 @@ var spec = transactions.SpecialAddresses{
 }
 
 func init() {
-	useEd25519Consensus := crypto.RandUint64()%2 == 0
+	useEd25519Consensus := true // pinned by benchspotter
 	fmt.Printf("Using Ed25519ConsensusBatchVerifier: %v\n", useEd25519Consensus)
 	crypto.SetEd25519BatchVerifier(useEd25519Consensus)
 }
